src/main/java/org/folio/rest/impl/OaiPmhSetImpl.java

```diff
@@ -91,12 +95,32 @@ public void deleteOaiPmhSetsById(String id, String lang, Map<String, String> oka
       Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     vertxContext.runOnContext(v -> {
       try {
+        logger.info("Delete set by id with id: '{}'", id);
         setService.deleteSetById(id, getTenantId(okapiHeaders))
           .map(DeleteOaiPmhSetsByIdResponse.respond204())
           .map(Response.class::cast)
           .otherwise(ExceptionHelper::mapExceptionToResponse)
           .onComplete(asyncResultHandler);
       } catch (Exception e) {
+        logger.error("Error occurred while posting set with id: '{}'. Message: {}. Exception: {}", id, e.getMessage(), e);
```

Comment:
not posting - deleting

Reply:
> not posting - deleting

I don't think so, there are will be changes relatively to metadata soon, it will be leaded to the common metadata strucutre as it used on the project. We don't want to save it as nested json object to database (I mean the json field type) cause we have to implement seraching mechanism which uses metadata fields and this won't be optimize to parse this json within searching logic.

Reply:
Thank you for note. Corrected